docs/how-to/integrate_oai_ran_with_observability.md

```diff
@@ -0,0 +1,128 @@
+# Integrate Charmed OAI RAN with Canonical Observability Stack
+
+[Charmed OAI RAN Terraform modules][Charmed OAI RAN Terraform modules] come with built-in support for the Canonical Observability Stack (COS).
+By default, COS deployment and integration is disabled.
+
+Each Charmed OAI RAN module exposes a set of configuration variables for deploying and integrating COS.
+
+This guide covers three ways of integrating Charmed OAI RAN with COS:
+1. [Integrating Charmed OAI RAN with COS at the deployment stage](#option-1)
+2. [Adding COS to an existing Charmed OAI RAN deployment](#option-2)
```

Comment:
I would remove those 2 options, I do not think we want to maintain this long term. With the adoption of CC006, we will drop support for managing the deployment of COS directly anyway.

Reply:
Done.